deep_translator/microsoft.py

```diff
@@ -80,7 +80,8 @@ def translate(self, text: str, **kwargs) -> str:
         # I have not added multiple text processing here since it is covered by the translate_batch method
         response = None
         if is_input_valid(text):
-            self._url_params["from"] = self._source
+            if self._source != 'auto':
```

Comment:
@Jourdelune

Reply:
hi, is it because if we don't do self._url_params["from"], the source argument in the __init__ method will not be used. That can be usefull for people that use custom detector to specify the language.

Reply:
Well what happens if source = auto? then the params will not be set? don't you think that would break the code

Reply:
yes it's stupid i don't know how i couldn't think of it sorry.
Sincerely.

Reply:
No worries. 
So how does region = global help? What is exactly you fixed here? 
Can you add more documentation to your PR please

Reply:
@Jourdelune Any news about this?